settings.sh

```diff
@@ -37,5 +37,5 @@ export BINARY HBREPO MAINT VENDOR DESC GOLANGCI_LINT_ARGS CONFIG_FILE
 export LICENSE FORMULA SOURCE_URL VERSION_PATH SOURCE_PATH
 
 # Optional
-export WINDOWS_LDFLAGS="-H windowsgui"
```

Comment:
This is because some of the libraries this app uses make exec calls that create quick flashing windows. Those suck. Doing it this way makes the app have a console window when it opens, but all sub-exec process use the same console window rather than have their own. We immediately hide that window, and no others appear. So there is now a flash of a window when launching this app and that is all.